keyboards/getta25/info.json

```diff
@@ -0,0 +1,39 @@
+{
+    "keyboard_name": "Getta25", 
+    "url": "https://salicylic-acid3.hatenablog.com/", 
+    "maintainer": "Salicylic_acid3", 
+    "width": 5.25, 
+    "height": 6.25, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                 {"label":"ESC", "x":0, "y":0},
+                 {"label":"F2", "x":1, "y":0},
+                 {"label":"=", "x":2, "y":0},
+                 {"label":"DEL", "x":3, "y":0},
+                 {"label":"Num Lock", "x":0, "y":1.25},
+                 {"label":"/", "x":1, "y":1.25},
+                 {"label":"*", "x":2, "y":1.25},
+                 {"label":"-", "x":3, "y":1.25},
+                 {"label":"7", "x":0, "y":2.25},
+                 {"label":"8", "x":1, "y":2.25},
+                 {"label":"9", "x":2, "y":2.25},
+                 {"label":"+", "x":3, "y":2.25, "h":2},
```

Comment:
Your layout macro in `getta25.h` is in a different order. We need to move some of these objects around.

```suggestion
```